lib/inets/src/http_server/httpd.erl

```diff
@@ -837,6 +839,7 @@ The fields of record `mod` have the following meaning:
 %%%========================================================================
 -type property() :: atom().
 -type ets_table() :: ets:tid().
+-type socket_type() :: ip_comm | ssl.
```

Comment:
I am not sure if we ought to have this alias for the ets:tid type although it has a better name! We can take it up one next scrum meeting. What do you think @erlang/team-ps

Reply:
The teams have never been used for this purpose. We can discuss doing that, but right now it does not work.

Reply:
what is the intended use for `erlang/team-ps`?